docs/usage.md

```diff
@@ -50,7 +50,9 @@ TODO
 TODO 
 
 ## Generate TypeScript models
-TODO 
+There are one way to generate models from object.
+
+- [Generate Data Model](../examples/generate-typescript-models)
```

Comment:
Maybe something along these lines is better:

```suggestion
Among many of the output languages we support, TypeScript is one of them. You can check out the [TypeScript documentation for more advanced use-cases](./languages/TypeScript.md).

Check out this [example out for a basic live demonstration](../examples/generate-typescript-models).
```

Thoughts @md-saif-husain ?

Reply:
@jonaslagoni : Thanks for feedback . Above suggestion make sense . I will update the usage.md

Reply:
@jonaslagoni : I have updated usage.md as per your suggestion